flecsi/coloring/mpi_communicator.h

```diff
@@ -586,11 +606,11 @@ class mpi_communicator_t : public communicator_t
       size_t ghost;
     }; // struct size_info_t
 
-    size_info_t buffer[colors];
+    std::unique_ptr<size_info_t[]> buffer(new size_info_t[colors]);
```

Comment:
Well, it's better as it is valid C++. The variable sized array is not valid C++, it's just supported by gcc/clang as an extension to be compatible with C99.

Reply:
Oh, of course&mdash;I just assumed that `colors` was constant here because it "had to be".  `std::vector` might be more typical, even if slightly less efficient, here.